guide.md

```diff
@@ -1399,31 +1399,41 @@ The only way to mitigate this is to encrypt your data on your side and then only
 
 ## Your Browser and Device Fingerprints:
 
-Your Browser and Device Fingerprints[^382] are set of properties/capabilities of your System/Browser. These are used on most websites for invisible user tracking but also to adapt the website user experience depending on their browser. For instance, websites will be able to provide a "mobile experience" if you are using a mobile browser or propose a specific language/geographic version depending on your fingerprint. Most of those techniques work with recent Browsers like Chromium-based[^251] browsers (such as Chrome/Edge) or Firefox[^252] unless taking specific measures.
+Your browser and device fingerprints[^382] are sets of properties/capabilities of your system/browser which can be analyzed. These are used on most websites for invisible user tracking, but can also be used to adapt the user experience (UX) depending on their browser type (e.g., mobile browsers). Some websites will be able to provide a mobile experience if you are using a mobile browser, or propose a specific language/geographic version depending on your fingerprint, your browser’s user agent, or your request headers. Most of those techniques work with recent browsers like Chromium-based browsers[^251] (such as Chrome and Edge) or Firefox[^252], unless the user takes specific precautions.
 
 You can find a lot of detailed information and publications about this on these resources:
 
--   <https://amiunique.org/links> <sup>[[Archive.org]](https://web.archive.org/web/https://amiunique.org/links)</sup>
+-   <https://amiunique.org/links> <sup>[[Archive.org]](https://web.archive.org/web/20220814180236/https://amiunique.org/links)</sup>
 
--   <https://brave.com/brave-fingerprinting-and-privacy-budgets/> <sup>[[Archive.org]](https://web.archive.org/web/https://brave.com/brave-fingerprinting-and-privacy-budgets/)</sup>
+-   <https://uniquemachine.org/> <sup>[[Archive.org]](https://web.archive.org/web/20220715074819/https://uniquemachine.org/)</sup>

```

Comment:
> (yes it seems we have two "your browser and device fingerprints" AND "your device and browser fingerprinting")

One is more the theory behind it, the other is the methods used. This can be clarified.

Reply:
We can also just maybe move the two sections to one (merge them) to save space after it's clarified (theoretically this is what FPing is, then explain how to mitigate).